wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -186,17 +186,13 @@ instance (HasSwagger subApi) => HasSwagger (WalletRequestParams :> subApi) where
 
 instance ToParamSchema WalletId
 
-instance ToSchema Core.Address where
-    declareNamedSchema = pure . paramSchemaToNamedSchema defaultSchemaOptions
-
 instance ToParamSchema Core.Address where
```

Comment:
:no_mouth:  Why exactly? Is this not used?

Reply:
:/ good catch, looks like a mistake